submissions/olabodeAmusan/JavaScript/index1.js

```diff
@@ -0,0 +1,11 @@
+for(let i = 1; i <= 123-7; i++){
```

Comment:
i <= 123-7

123-7 can be identified with a variable. 

This code should aim at been reusable

Reply:
Done. Please review